<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(166, 74, 74);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="predict_pokemon">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 'light gray';
font: 18pt &quot;Comic Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Predict Pokemon!</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text_input">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 'white';</string>
    </property>
    <property name="cursorWidth">
     <number>21</number>
    </property>
    <property name="placeholderText">
     <string>Insert Pokémon Description Here...</string>
    </property>
   </widget>
   <widget class="QLabel" name="first_pokemon">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="second_pokemon">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>590</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="third_pokemon">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>640</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="first_pokemon_border">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>GUI_images/gold_border_2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="second_pokemon_border">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>560</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>GUI_images/silver_border.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="third_pokemon_border">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>592</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>GUI_images/bronze_border.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ash_pretext">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>431</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>GUI_images/ash_pretext_image_1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="ash_text">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Describe a &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Pokémon in&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;the text box!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="first_num">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 italic 32pt &quot;Avenir&quot;;</string>
    </property>
    <property name="text">
     <string>1st: </string>
    </property>
   </widget>
   <widget class="QLabel" name="second_num">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>770</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 italic 32pt &quot;Avenir&quot;;</string>
    </property>
    <property name="text">
     <string>2nd:</string>
    </property>
   </widget>
   <widget class="QLabel" name="third_num">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>800</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 italic 32pt &quot;Avenir&quot;;</string>
    </property>
    <property name="text">
     <string>3rd:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ash_posttext">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>431</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>GUI_images/ash_posttext_image.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="i_choose_you">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;I CHOOSE YOU!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="first_place_name">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="second_place_name">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>770</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="third_place_name">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>800</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>431</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>GUI_images/pykedex_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>predict_pokemon</zorder>
   <zorder>text_input</zorder>
   <zorder>first_pokemon_border</zorder>
   <zorder>first_pokemon</zorder>
   <zorder>third_pokemon_border</zorder>
   <zorder>third_pokemon</zorder>
   <zorder>ash_pretext</zorder>
   <zorder>second_pokemon_border</zorder>
   <zorder>second_pokemon</zorder>
   <zorder>ash_text</zorder>
   <zorder>first_num</zorder>
   <zorder>second_num</zorder>
   <zorder>third_num</zorder>
   <zorder>ash_posttext</zorder>
   <zorder>i_choose_you</zorder>
   <zorder>first_place_name</zorder>
   <zorder>second_place_name</zorder>
   <zorder>third_place_name</zorder>
   <zorder>logo</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1109</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="GUI_images/background_resource.qrc"/>
 </resources>
 <connections/>
</ui>
